openreview/journal/process/ae_assignment_process.py

```diff
@@ -51,9 +70,9 @@ def process_update(client, edge, invitation, existing_edge):
         ## expire AE recommendation
         journal.invitation_builder.expire_invitation(journal.get_ae_recommendation_id(number=note.number))
 
-        ## update assigned_action_editor if exists in the submission
+        ## add assigned_action_editor
         content = {}
-        if 'assigned_action_editor' in note.content:
+        if note.content['venueid']['value'] == journal.under_review_venue_id:
```

Comment:
Will an AE be added to the submission only when the `venueid == journal.under_review_venue_id`? Could the venue id be `journal.assigning_AE_venue_id` instead and need the `assigned_action_editor` added?

Reply:
usually `assigned_action_editor` is being set when the paper is under review.

I'm handling this case when the AE is unassigned and then another one is assigned when the paper is under review.

Reply:
BTW I will make a new PR where the assigned_action_editor will be visible as soon as the AE is assigned.